controllers/cart/addCart.js

```diff
@@ -19,7 +21,20 @@ async function addCart(req, res) {
     cart.storeId = store._id
     const newCart = new Cart(cart)
     const result = await newCart.save()
-    return res.status(200).json(result)
+    const link = 'http://' + req.headers.host + '/api/store/cart/'// to be changed to FE route
+    const msg = {
+      to: `${result.email}`,
+      from: process.env.FROM_EMAIL,
+      subject: 'Pure Retail email notification for your saved cart',
+      text: 'This link will lead you to your saved cart',
+      html: `<strong><a href=${link}>Click me</a></strong>`
+    }
+    await sgMail.send(msg)
+
+    res.status(200).json({
+      status: 'A reset phone has been sent to ' + result.email + '.',
```

Comment:
i think the status message needs to be restructured

Reply:
Ok, I have made some changes to the status message.